content/manager_webui/overview.md

```diff
@@ -61,3 +61,11 @@ The UI templates are the sets of pages presented to the users upon logging into
 - Local Blueprint Catalog
 - Deployments
 - System Resources
+
+## Advanced Configuration
+
+In `/opt/cloudify-stage/dist/userData/userConfig.json` on Cloudify Manager machine you can perform more advanced customization of Cloudify Console.
+
+List of possible settings and its default values can be found in [cloudify-cosmo/cloudify-stage repository](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/conf/userConfig.json).
```

Comment:
List of possible settings and its default values can be found in >
You can see a list of advanced settings and values in

Reply:
Done.